aspnet/1-hello-world/runTests.ps1

```diff
@@ -1,14 +1,15 @@
-﻿# Copyright(c) 2016 Google Inc.
-#
-# Licensed under the Apache License, Version 2.0 (the "License"); you may not
-# use this file except in compliance with the License. You may obtain a copy of
-# the License at
-#
-# http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
-# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
-# License for the specific language governing permissions and limitations under
-# the License.
+﻿# Copyright(c) 2016 Google Inc.
```

Comment:
For reasons unknown, Powershell editor saved all the .ps1 files as unicode instead of text.  The only real diff is the new line:
BuildSolution